.github/workflows/release-artifacts.yml

```diff
@@ -91,7 +91,34 @@ jobs:
 
   build-sdist:
     name: "Build pypi distribution files"
-    uses: "matrix-org/backend-meta/.github/workflows/packaging.yml@v1"
+    steps:
```

Comment:
should whatever changed here be folded back into the backend-meta repo?

Reply:
Ah yup, probably. The change is using `poetry build` rather than `pip build`